apps/gruenbuch/models.py

```diff
@@ -1,3 +1,82 @@
 from django.db import models
+from wagtail.admin.edit_handlers import FieldPanel
+from wagtail.admin.edit_handlers import ObjectList
+from wagtail.admin.edit_handlers import StreamFieldPanel
+from wagtail.admin.edit_handlers import TabbedInterface
+from wagtail.core import blocks, fields
+from wagtail.core.models import Page
 
-# Create your models here.
+from apps import blocks as apps_blocks
+from apps.contrib.translations import TranslatedField
+
+class GruenbuchIndexPage(Page):
+
+    subpage_types = ['apps_gruenbuch.GruenbuchDetailPage']
+
+class GruenbuchDetailPage(Page):
+
+    page_blocks = [
+        ('paragraph', blocks.RichTextBlock()),
+        ('faq_accordion', apps_blocks.FaqBlock())
+    ]
+
+    page_title_de = models.CharField(max_length=120, blank=True)
```

Comment:
I think we should require the page_title_de and body_de, as German is the fallback language and the page doesn't make sense without it.
For subtitle, we can then ask `{% if page.subtitle %}` and only display it in that case.